server/src/controllers/Venue/resolver.ts

```diff
@@ -102,6 +104,48 @@ export class VenueResolver {
     @Arg('_onlyUsedForAuth', () => Int) _onlyUsedForAuth: number,
   ): Promise<{ id: number }> {
     // TODO: handle deletion of non-existent venue
+    const users = await prisma.users.findMany({
+      where: {
+        user_events: {
+          every: {
+            event: {
+              canceled: false,
+              ends_at: { gt: new Date() },
+              venue_type: events_venue_type_enum.Physical,
+            },
+          },
+        },
+      },
+    });
```

Comment:
This seem to find much more than it's needed. `venue_type` don't really matter when we can just look for the `venue_id`. Also, the `every` would require every `user_events` to match the conditions, to find specific user, while `some` needs just one.
```suggestion
    const users = await prisma.users.findMany({
      where: {
        user_events: {
          some: {
            event: {
              canceled: false,
              ends_at: { gt: new Date() },
              venue_id: id,
            },
            subscribed: true,
          },
        },
      },
    });
```